api.go

```diff
@@ -409,7 +415,13 @@ func (api *API) call(baseAPI string, endpoint string, body interface{}, out inte
 		return err
 	}
 
-	targetURL := fmt.Sprintf("%s/v1/%s/%s", api.BaseURL, baseAPI, endpoint)
+	var targetURL string
+	if baseAPI == "wallet" && api.WalletURL != "" {
```

Comment:
Can the wallet API be located at the same location as the `BaseURL` or it's impossible? If the latter is true (cannot be at same location than BaseURL), than we should probably return an error if somehow is trying to use the `wallet` baseAPI but did not configure correctly `WalletURL`.